### PR TITLE
refine Nano setup directions

### PR DESCRIPTION
**Pull request by aaronmarkham**
## Description ##
After another run-through on a fresh Nano, plus trying out these new pip wheels, we found some refinements that should make the setup go a little more cleanly.

Thanks for the assist @darlaker

### CODE REVIEW

**Comment by aaronmarkham** [comment]
Nudging CI again...

File: docs/install/install-jetson.md [hunk 1]
```diff
@@ -60,49 +59,54 @@ sudo pip install \
 ```
 
 If you plan to cross-compile you will need to install these dependencies on that computer as well.
+If you get an error about something being busy, you can restart the nano and this error will go away and you can continue installation of the prerequisites.
 
-### Configure CUDA
+### Download the source & setup some environment variables:
 
-You can check to see what version of CUDA is running with `nvcc`.
+These steps are optional, but some of the following instructions expect MXNet source files and the `MXNET_HOME` environment variable. Also CUDA commands will not work out of the box without updating your path.
```
Comment (IvyBazan) [inline docs/install/install-jetson.md]: "Also, CUDA commands will not work out of the box without updating your path."

File: docs/install/install-jetson.md [hunk 2]
```diff
@@ -128,12 +130,13 @@ To use this with the MXNet Python binding, you must match the source directory's
 ```bash
 cd $MXNET_HOME
 git checkout v1.4.x
-git submodule update --init
+git submodule update --init --recursive
 cd python
 sudo pip install -e .
 ```
```
Comment (darlaker) [inline docs/install/install-jetson.md]: I think we considered adding a note here at the end of Option 1 that notifies the user that at this point, they are done and do not need to continue. I think that still sounds helpful, but that may already be clear with "Option" now in the title.

File: docs/install/install-jetson.md [hunk 3]
```diff
@@ -60,49 +59,54 @@ sudo pip install \
...
 
-Setup your environment variables for MXNet.
+To switch CUDA versions on a device or computer that has more than one version installed, use the following and replace the version as appropriate.
 
 ```bash
-cd ..
-export MXNET_HOME=$(pwd)
-echo "export PYTHONPATH=$MXNET_HOME/python:$PYTHONPATH" >> ~/.rc
-source ~/.rc
+sudo rm /usr/local/cuda
```
Comment (darlaker) [inline docs/install/install-jetson.md]: Maybe add a line here that describes how these commands will remove the directory and then replace it by creating a symlink. But Linux users may not need this called out. :)

File: docs/install/install-jetson.md [hunk 4]
```diff
@@ -60,49 +59,54 @@ sudo pip install \
...
+You can check to see what version of CUDA is running with `nvcc`.
 
 ```bash
-git clone --recursive https://github.com/apache/incubator-mxnet.git mxnet
-cd mxnet
+nvcc --version
 ```
 
-Setup your environment variables for MXNet.
+To switch CUDA versions on a device or computer that has more than one version installed, use the following and replace the version as appropriate.
```
Comment (darlaker) [inline docs/install/install-jetson.md]: Maybe add: "This example installs version 10 and replaces all other versions."

File: docs/install/install-jetson.md (hunk 3 above)
Comment (aaronmarkham) [reply docs/install/install-jetson.md]: It was always a symlink, not an actual folder. I just briefly freaked out when I thought I deleted the cuda files.

**Pull request merged**